src/devices/lcec_deasda.c

```diff
@@ -191,42 +259,56 @@ static int lcec_deasda_init(int comp_id, struct lcec_slave *slave, ec_pdo_entry_
   // set interpolation time period
   tu = master->app_time_period;
   ti = -9;
+
   while ((tu % 10) == 0 || tu > 255) { tu /=  10; ti++; }
   if (lcec_write_sdo8(slave, 0x60C2, 0x01, (uint8_t)tu) != 0) {
     rtapi_print_msg (RTAPI_MSG_ERR, LCEC_MSG_PFX "fail to configure slave %s.%s sdo ipol time period units\n", master->name, slave->name);
   }
   if (lcec_write_sdo8(slave, 0x60C2, 0x02, ti) != 0) {
     rtapi_print_msg (RTAPI_MSG_ERR, LCEC_MSG_PFX "fail to configure slave %s.%s sdo ipol time period index\n", master->name, slave->name);
-  }
```

Comment:
I *think* this `}` needs to stay in the file; that's what's causing the build failure.

Reply:
Your thinking was right;). Also, during the merge another bad bad occurred that disabled CSP mode. Fixed and tested.